### PR TITLE
Update config

### PR DESCRIPTION
**Pull request by king-11**
- add python version and  pyenv instructions
- use pyproject.toml and poetry for virtualenv creating
- remove heroku config and use dj database URL ourself
- update readme with instructions for local setup, deployment and super user creation

### CODE REVIEW

File: todo/settings.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 """
 
 import os
-import django_heroku
```
Comment (nishantwrp) [inline todo/settings.py:14]: I miss `django_heroku` 😞.

Comment (king-11) [reply todo/settings.py:14]: 😢 had to do so much of new config just to makeup for heroku

Comment (krashish8) [reply todo/settings.py:14]: > I miss django_heroku 😞.

+1 🥲

**Review by krashish8 — APPROVED** [review]
Thanks for the update! 🙌

**Pull request merged**